packages/playground-ui/theme.css

```
@@ -71,6 +71,24 @@
   --notice-note: oklch(0.18 0 0);
   --notice-note-fg: oklch(0.99 0.001 230);
 
+  --badge-neutral-fg: var(--neutral5);
+  --badge-green: oklch(0.8 0.105 160);
+  --badge-green-fg: oklch(0.92 0.05 160);
+  --badge-red: oklch(0.69 0.24 22);
```

Stylelint (17.14.0)

[error] Expected "22" to be "22deg" (hue-degree-notation)

(hue-degree-notation)